### ISSUE TITLE
No documentation about deploying ASP.NET for "Debian, Ubuntu and derivatives"

### ISSUE THREAD

**Issue by sanecito** [aspnet/Docs#399]
Documentation is provided on how to install Mono and its dependencies and other required components at:

 http://docs.asp.net/en/latest/getting-started/installing-on-linux.html?highlight=mono#installing-on-debian-ubuntu-and-derivatives

However, there is no documentation about deploying ASP.NET after everything is installed on the aforementioned page or on the deployment page (http://docs.asp.net/en/latest/publishing/index.html) . For example, do I copy my project folder, "[...]\Documents\Visual Studio 2015\Projects\MvcMovie" sans solution file to a particular sub-directory of Mono is housed in, etc after the project is built using default settings?

**Comment by SpaceShot** [comment]
I am willing to add this documentation if needed.  I did use it as inspiration for this blog post and the automated script I developed.  The Ubuntu script, imho, is in good shape (but there is some tinkering I do want to do).

I don't speak for the aspnet team of course and there may be a very good reason they stop at dnvm installation.

**Comment by shirhatti** [comment]
This seems to be a duplicate of https://github.com/aspnet/Docs/issues/106
We are currently working on a production guidance for Linux and I hope to have PR out soon. 

@SpaceShot Feel free to review and give feedback on the PR that I will put out soon

**Comment by sanecito** [comment]
Yeah, it does look that way, sorry for the duplicate issue; didn't see it when searching. Feel free to close it and label it as a duplicate.

@SpaceShot I actually think I may go the Docker route. When I heard about .NET going open source last year all of the articles I read Mono was named dropped in, so I figured when I came to a project recently that required writing an ASP.NET app which would be run a *nix server running Apache, I was deadset on using Mono as part of my solution as I thought that's what was required. There's loads of (readable) documentation that I've found regarding Docker particularly the simple, but detailed install & deploy doc MS has at http://blogs.msdn.com/b/webdev/archive/2015/01/14/running-asp-net-5-applications-in-linux-containers-with-docker.aspx That said, documentation of deploying using Mono instead of Docker isn't as much of a pressing issue for me now, so I probably won't be utilizing your script/article. It looks cool though and hopefully you and other dev's can push the non-Docker solution soon for other folk. :+1:
